huaweicloud/services/cse/default_values.go

```diff
@@ -0,0 +1,3 @@
+package cse
+
+var CseVserion = "CSE2"
```

Comment:
suggest rename it to **DefaultVersion**, then we can import is as ces.DefaultVersion

Reply:
fixed and moved to resource file